src/main/java/com/compilercharisma/chameleonbusinessstudio/controller/UserController.java

```diff
@@ -41,7 +42,7 @@ public Mono<User> getUser(@PathVariable String email) {
      *
      * @return {@link UserResponse}
      */
-    @GetMapping() // implies "all"
+    @GetMapping("/getAllUsers")
```

Comment:
why are you changing the endpoints back?

Reply:
My bad, when I merged from develop I didn't notice you actually got rid of them and so I put them back, this should be like it was now.

Reply:
Oh duh, merge conflicts. Makes sense now, I just thought this branch involved changes from mine. Sorry!